website/docs/guides/1.0.0-upgrade-guide.html.markdown

```diff
@@ -79,6 +79,155 @@ so no changes are needed.
 **NOTE** Doc links for [mongodbatlas_privatelink_endpoint](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint)
 **NOTE** Doc links for [mongodbatlas_privatelink_endpoint_service](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_service)
 
+## Migration to Terraform SDK v2
+
+Because of migration to terraform sdk v2, in various resources and datasource(s) are not supported using TypeMap as object, instead it should use TypeSet or TypeList with MaxItems:1, here the list of resources affected by migration
```

Comment:
Just a rewrite to make a bit more clear, please replace this with:
Due to the migration to Terraform SDK v2 various resources and datasource(s) have been impacted and will require a change within your configuration.  These resources and datasources used the now unsupported TypeMap object.  In SDK v2 these must be TypeSet or TypeList with MaxItems set to 1.    The following is the list of resources and datasources impacted by the migration.